docs/source/bentocloud/how-tos/manage-deployments.rst

```diff
@@ -40,41 +40,58 @@ To retrieve details about a specific Deployment:
       # To output the details in YAML (Default)
       bentoml deployment get <deployment-name> -o yaml
 
+    Expected output in YAML:
+
+    .. code-block:: yaml
+
+        name: summarization
+        bento: summarization:ghfvclwp2kwm5e56
+        cluster: aws-ca-1
+        endpoint_url: https://summarization-test--aws-ca-1.mt1.bentoml.ai
+        admin_console: https://test.cloud.bentoml.com/deployments/summarization/access?cluster=aws-ca-1&namespace=test--aws-ca-1
+        created_at: '2024-02-20 09:27:52'
+        created_by: bentoml-user
+        config:
+          envs: []
+          services:
+            Summarization:
+              instance_type: cpu.2
+              scaling:
+                min_replicas: 1
+                max_replicas: 2
+              envs: []
+              deployment_strategy: Recreate
+              extras: {}
+              config_overrides:
+                traffic:
+                  timeout: 10
+        status:
+          status: running
+          created_at: '2024-02-20 09:27:52'
+          updated_at: '2024-02-21 05:46:18'
+
   .. tab-item:: Python API
 
     .. code-block:: python
 
-      bentoml.deployment.get(name="deployment-1")
-
-Expected output in YAML:
-
-.. code-block:: yaml
-
-    name: summarization
-    bento: summarization:ghfvclwp2kwm5e56
-    cluster: aws-ca-1
-    endpoint_url: https://summarization-test--aws-ca-1.mt1.bentoml.ai
-    admin_console: https://test.cloud.bentoml.com/deployments/summarization/access?cluster=aws-ca-1&namespace=test--aws-ca-1
-    created_at: '2024-02-20 09:27:52'
-    created_by: bentoml-user
-    config:
-      envs: []
-      services:
-        Summarization:
-          instance_type: cpu.2
-          scaling:
-            min_replicas: 1
-            max_replicas: 2
-          envs: []
-          deployment_strategy: Recreate
-          extras: {}
-          config_overrides:
-            traffic:
-              timeout: 10
-    status:
-      status: running
-      created_at: '2024-02-20 09:27:52'
-      updated_at: '2024-02-21 05:46:18'
+      import bentoml
+
+      dep = bentoml.deployment.get(name="deploy-1")
+      print(dep)
+
+      # Output: DeploymentInfo(name='deploy-1', admin_console='https://test.cloud.bentoml.com/deployments/deploy-1/access?cluster=aws-ca-1&namespace=test--aws-ca-1', created_at='2024-03-01 05:00:19', created_by='bentoml-user', cluster='aws-ca-1')
+
+    To check the Deployment's status:
+
+    .. code-block:: python
+
+      import bentoml
+
+      dep = bentoml.deployment.get(name="deploy-1")
+      status = dep.get_status()
```

Comment:
Should we explicitly show all the valid method and attributes that `dep` have?

Also,  get_status has one parameter, refetch, like: `dep.get_status(refetch=True)`. By default, refetch is True, but user can also set it to False

Reply:
I suggest we only list the common ones, but depending on their purpose, they may be covered in different docs. So in addition to `get_status()`, what are other methods or attributes we can use for `dep`?

Reply:
I added some paramaters and methods. Please take a look again. Thanks.